tutorials/marble/explore-the-network.md

```diff
@@ -0,0 +1,46 @@
+# Explore the Network
+
+```{contents}
+:local:
+```
+
+## Learn about the network
```

Comment:
I think this heading should be removed since this is an introductory paragraph. Otherwise we have a H1 heading "Explore the Network" and right under, a H2 heading "Learn about the network" which sounds redundant.